nemo/src/io/formats/rdf/reader.rs

```diff
@@ -184,7 +170,7 @@ impl RdfReader {
         let stop_limit = self.limit.unwrap_or(u64::MAX);
         let triple_count = Cell::new(0);
 
-        let mut on_triple = |triple: Triple| {
+        let mut on_triple = |triple: Quad| {
```

Comment:
Just double checking: `Triple` is `Quad` now?

Reply:
Yes, `RdfParser` always iterates over Quads, since it doesn't know up front which format it will parse.

Reply:
Ok, then I think we should also rename variables accordingly to prevent confusion.

Reply:
I hope this is clearer now.

Reply:
Thanks!